ProjectLighthouse.Servers.Website/Controllers/Admin/ModerationSlotController.cs

```diff
@@ -27,6 +28,16 @@ public async Task<IActionResult> TeamPick([FromRoute] int id)
         if (slot == null) return this.NotFound();
```

Comment:
GitHub doesn't let me mark the correct line but on line 27 it should be 
`Slot? slot = await this.database.Slots.Include(s => s.Creator).FirstOrDefaultAsync(s => s.SlotId == id);`
Note the Include() call to fetch the User foreign key, after that you should be able to use slot.Creator.Username in the webhook

Reply:
Fixed